.github/workflows/update-ss3-version.yml

```diff
@@ -71,4 +72,5 @@ jobs:
 
            Auto-generated by [update-ss3-version.yml][1]. Binaries from ${{ steps.get-link-to-release.outputs.link_to_release }}.
 
-            [1]: https://github.com/nmfs-stock-synthesis/workflows/tree/main/.github/workflows/update-ss3-version.yml
```

Comment:
We will not need to test this workflow out because we already know it works. So, these edits can be removed.